non_profit_organization/data/base_automation.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>
+    <record id="base_automation_1" model="base.automation">
+        <field name="name">Update pricelist of customer with ongoing membership subscription</field>
+        <field name="model_id" ref="sale.model_sale_order"/>
+        <field name="action_server_ids" eval="[(6, 0, [ref('non_profit_organization.ir_act_server_711')])]"/>
+        <field name="filter_domain">["|", ("subscription_state", "=", "3_progress"), ("subscription_state", "=", "5_renewed")]</field>
+        <field name="filter_pre_domain">["&amp;", ("plan_id", "!=", False), ("order_line.product_id.name", "ilike", "member")]</field>
```

Comment:
Use XML ID instead of name and add it after the product in manifest
```suggestion
        <field name="filter_pre_domain" evel="['&amp;', ('plan_id', '!=', False), ('order_line.product_id.id', '=', ref('product_product_1'))]</field>
```